lib/client-assets.php

```diff
@@ -370,7 +370,7 @@ function gutenberg_register_packages_styles( $styles ) {
 		$styles,
 		'wp-reset-editor-styles',
 		gutenberg_url( 'build/block-library/reset.css' ),
-		array(),
+		array( 'common', 'forms' ), // Make sure the reset is loaded after the default WP Adminn styles.
```

Comment:
This dependency causes the admin styles to be added within the iframe. Any way we can avoid that?

Reply:
What is adding these styles on the front-end? Surely not `common.css`? Can we use the same stylesheet as the front-end?

Reply:
The reason we added these dependencies here is because the "reset" style need to be loaded after "common" and "forms" otherwise you get this kind of issues https://github.com/WordPress/gutenberg/pull/30141#issuecomment-808369289 

--
I'm not really sure how this relates to the iframe though because all these three stylesheets (reset, common and forms) are probably better off of the iframe (the reset style is only needed because of WP-Admin default styles). 

I'm also not sure about your question about frontend, this is unrelated to frontend but in order for the editor to match frontend we need to "reset" some WP-Admin styles inside the canvas (in non-iframe mode) and that's what this reset style does.